pkg/workloads/console/integration/integration_test.go

```diff
@@ -85,12 +90,21 @@ var _ = Describe("Console", func() {
 			HaveOccurred(), "failed to create Console",
 		)
 
-		By("Expect reconcile succeeded")
-		Eventually(calls, timeout).Should(
-			Receive(
-				integration.ReconcileResourceSuccess(namespace, "console-0"),
-			),
-		)
+		waitForSuccessfulReconcile = func(times int) {
```

Comment:
This seems a bit ugly, but worked for what I needed. It'd be nice to perhaps make it a bit more general and put it in the `integration` package, but I propose that if the reviewer agrees then we create an issue for that, rather than implement in this PR.

Reply:
If this were rspec we'd write  something like
```go
Eventually(calls, timeout).Should(Receive(...).Exactly(n).Times())
```
I don't think gomega supports that, but it would be a nice way of writing it.